wbia/control/manual_part_funcs.py

```diff
@@ -721,7 +721,7 @@ def get_part_qualities(ibs, part_rowid_list, eager=True):
         tbl = part
 
     SeeAlso:
-        wbia.const.QUALITY_INT_TO_TEXT
```

Comment:
We should replace all of these instances with `wbia.constants.` since it is documentation